powersoftau/src/raw/raw_accumulator.rs

```diff
@@ -267,7 +285,9 @@ pub fn verify<E: PairingEngine>(
                 };
 
                 rayon::scope(|t| {
+                    let _enter = span.enter();
                     t.spawn(|_| {
+                        let _enter = span.enter();
```

Comment:
let _ =?

Reply:
This is the recommended way from `tracing` docs